news/colorupdate.rst

```diff
@@ -4,6 +4,8 @@
 
 **Changed:**
 
+* <news item>
```

Comment:
It's added back. @sbillinge

Reply:
<img width="783" alt="Screenshot 2024-08-20 at 9 25 40 PM" src="https://github.com/user-attachments/assets/da47306b-8465-4df9-98b5-50dab676bf79">

Again, CI works from my forked repo but not sure why it's failing here. Perhaps I am thinking it's due to cache